### PR TITLE
SSE: Add abort and closed methods

### PR DESCRIPTION
**Pull request by AlexanderKolberg**

### CODE REVIEW

File: _examples/sse/webapp/client.gen.ts [hunk 1]
```diff
@@ -549,8 +577,14 @@ export interface WebrpcStreamOptions<T> extends WebrpcOptions {
...
   onClose?: () => void;
 }
+
 export interface WebrpcOptions {
   headers?: HeadersInit;
   signal?: AbortSignal;
 }
 
+export interface WebrpcStreamController {
+  abort: (reason?: any) => void;
```
Comment (VojtechVitek) [inline _examples/sse/webapp/client.gen.ts:587]: can we call it `close()`, as in https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events ?

**Review by VojtechVitek — COMMENTED** [review]
Looks good :) One suggestion

Let's try it on the main webrpc repo

File: _examples/sse/webapp/client.gen.ts (hunk 1 above)
Comment (AlexanderKolberg) [reply _examples/sse/webapp/client.gen.ts:587]: I want to reserve the name "close" for a potential future resumable close event. What's happening here is triggering an abort event, so I think the name "abort" makes sense

**Comment by AlexanderKolberg** [comment]
> Looks good :) One suggestion
> 
> Let's try it on the main webrpc repo

https://github.com/webrpc/webrpc/pull/311

**Pull request merged**